android/modules/ui/src/java/ti/modules/titanium/ui/widget/picker/TiUINativePicker.java

```diff
@@ -41,7 +41,8 @@
 {
 	private static final String TAG = "TiUINativePicker";
 	private boolean firstSelectedFired = false;
-	private static int defaultTextColor = 0;
+	private static int defaultTextColor = -1;
+	private static boolean setDefaultColor = false;
```

Comment:
That would throw an error: you can't use boolean operators on integer types in Java.